envs/example/defaults-2.0.yml

```diff
@@ -197,6 +197,28 @@ ceph:
 lvm:
   enabled: false
 
+v7k:
```

Comment:
These variables are both in defaults-2.0.yml and storwize/defaults/main.yml.
As cinder role needs these variables, I think that's why you put them here.
Variables in storwize/defaults/main.yml are always overwritten by defaults-2.0.yml.
We may need to remove variables in storwize/defaults/main.yml.